tips_n_tricks/week3/map-collisions.md

```diff
@@ -0,0 +1,151 @@
+# Map Collisions
+
+## Review
+
+So quick review on collisions, we've covered:
+
+1. distance < half widths  
+![](distance_width.gif). 
+2. standard box-box collision (AABB)
+(gif coming soon)
+3. The pixel perfect [Duck Hunt](https://www.youtube.com/watch?v=-1NyIsZXeqU)-style detection (gif coming soon)
+
+\* here's [a cart (coming soon)](#) that the above gifs came from
+
+## Map Collisions
+
+Now how about colliding with what we have drawn on the map?  
+We could so for each moving object go through each map sprite and and do box-box collision on the two, but that'd be a lot of checks!  
+
+PICO-8 has some convenient functions that let us do collisions with the map a little more easily.  
+These functions are:
+
+* [`mget`](http://pico-8.wikia.com/wiki/Mget): gets the sprite number at the map coordinate given
+* [`fget`](http://pico-8.wikia.com/wiki/Fget): gets the flag(s) set on a sprite number
```

Comment:
So this felt a little weird reading it through the first time and I think I figured out why. The extra visual space dedicated to presenting mget and fget in a list is perhaps not worth it--inline with the introductory paragraph is fine. 

In addition, you introduce some context for each function here that may be confusing and could be omitted. You have dedicated sections to each, in any case. Let your sections introduce and expand upon the functionality.

Reply:
mm very useful feedback. I'll get to that tomorrow morning. Thank you!